Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -512,7 +512,7 @@ void NetworkStorageManager::performEviction(HashMap<WebCore::SecurityOriginData,
     }
 
     UNUSED_PARAM(deletedOriginCount);
-    RELEASE_LOG(Storage, "%p - NetworkStorageManager::performEviction evicts %" PRIu64 " origins, current usage %" PRIu64 ", total quota %" PRIu64, this, deletedOriginCount, valueOrDefault(m_totalUsage), m_totalQuota);
+    RELEASE_LOG(Storage, "%p - NetworkStorageManager::performEviction evicts %" PRIu64 " origins, current usage %" PRIu64 ", total quota %" PRIu64, this, deletedOriginCount, valueOrDefault(m_totalUsage), *m_totalQuota);
```

Comment:
You'd want `valueOrDefault` here, right? In case it's a null value

Reply:
There's a null check above, on line 487.